eye2bids/_base.py

```diff
@@ -48,6 +49,7 @@ def update_from_metadata(self, metadata: None | dict[str, Any] = None) -> None:
             "ScreenRefreshRate": metadata.get("ScreenRefreshRate"),
             "ScreenSize": metadata.get("ScreenSize"),
         }
+        self["TaskName"] = metadata.get("TaskName")
```

Comment:
OK what if the data is both in the metadata and the data file we are reading from: which one takes precedence?
I think we should decide and make that a rule for all similar cases.